cli/cmd/install.go

```diff
@@ -157,6 +157,7 @@ func validateAndBuildConfig(options *installOptions) (*installConfig, error) {
 		return nil, err
 	}
 
+	// TODO: these seem to not be used?
```

Comment:
@alpeb Would like your thoughts on this, is it expected? Is it related/resolved in #2334?